build.gradle

```diff
@@ -37,46 +62,37 @@ shadowJar {
     destinationDir = file("${buildDir}/jar/")
 }
 
-task myZip(dependsOn: 'npmRunSpuild', type: Zip) {
+task zipReport(dependsOn: 'npmRunSpuild', type: Zip) {
     from 'frontend/build/'
     baseName = 'templateZip'
     destinationDir = file('src/main/resources')
 }
 
```

Comment:
In addition to the change suggested by @yong24s , 
let's remove this new line as well so that the task and the dependency appears as a module.